commons-db/src/main/java/com/palantir/nexus/db/sql/BasicSQLUtils.java

```diff
@@ -53,7 +52,27 @@ public static final int countFields(String fieldList) {
     }
 
     public static String nArguments(int n) {
-        return TextUtils.makeStringList("?", ",", n); //$NON-NLS-1$ //$NON-NLS-2$
+        return makeStringList("?", ",", n); //$NON-NLS-1$ //$NON-NLS-2$
+    }
+
+    /**
+     *  Makes a list of n copies of the given string, separated by the given delimiter.
+     */
+    // TODO this method was copied from TextUtils, which is being moved to an internal product.
+    // If this class follows it there, then the duplication should be removed.
+    private static String makeStringList(String str, String delimiter, int count) {
```

Comment:
Maybe it's worth simplifying these now that they aren't in papaya? Like:
`return Joiner.on(',').join(Iterables.limit(Iterables.cycle('?'), n));`